terminal.go

```diff
@@ -304,6 +304,7 @@ func (g *LightningTerminal) startSubservers() error {
 			MacaroonDir:           macDir,
 			TLSPath:               tlsPath,
 			BlockUntilChainSynced: true,
+			BlockUntilUnlocked:    true,
```

Comment:
🎉 

One question here in the context of lit. Since we just log that we are waiting for lnd to be unlocked, I imagine there won't be any feedback indicating that the user needs to unlock? Still an improvement over crashing (which was the previous behaviour I think?), just wondering about it longer term.

Reply:
Yeah, you'd only see this in the log at the moment. This will be fully solved by implementing the unlock in the UI itself.